README.md

```diff
@@ -790,6 +790,24 @@ May also be configured in `lerna.json`:
 }
 ```
 
+#### --npm-client-args [client]
```

Comment:
This needs to be defined for yargs in the `builder` export of BootstrapCommand.

Even better, why not just accept all args after a `--` to `lerna bootstrap` to be passed to the client execution?

```
lerna bootstrap -- --ignore-engine --silent
```

To pass it via lerna.json, you'd need to pass an array of strings instead of a string:
```json
"npmClientArgs": ["--ignore-engine", "--silent"]
```